app/locales/ne.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
Is this file supposed to be named `ne.json` or should it be `np.json`?

Reply:
`np` is not in this list: https://github.com/ladjs/i18n-locales so I think it should be `ne.json`

Reply:
Ah sorry, now I remember :)